.github/workflows/auto-assignee.yml

```diff
@@ -0,0 +1,11 @@
+name: Add assignee to PRs
+on:
+  pull_request:
+    types: [ opened, reopened ]
+permissions:
+  pull-requests: write
+jobs:
+  assign-author:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: toshimaru/auto-author-assign@2daaeb2988aef24bf37e636fe733f365c046aba0
```

Comment:
issue: I believe this will overwrite if the PR is already assigned? Should it first check that there is not already an assignee?

Reply:
Ah it will skip in this condition
> Someone is already assigned as an assignee